actors/builtin/miner/miner_actor.go

```diff
@@ -215,19 +215,21 @@ func (a Actor) SubmitWindowedPoSt(rt Runtime, params *SubmitWindowedPoStParams)
 		if uint64(len(params.Partitions)) > submissionPartitionLimit {
 			rt.Abortf(exitcode.ErrIllegalArgument, "too many partitions %d, limit %d", len(params.Partitions), submissionPartitionLimit)
 		}
-		deadline := st.DeadlineInfo(currEpoch)
-		if !deadline.PeriodStarted() {
-			rt.Abortf(exitcode.ErrIllegalArgument, "proving period %d not yet open at %d", deadline.PeriodStart, currEpoch)
-		}
-		if deadline.PeriodElapsed() {
-			// A cron event has not yet processed the previous proving period and established the next one.
-			// This is possible in the first non-empty epoch of a proving period if there was an empty tipset on the
-			// last epoch of the previous period.
-			rt.Abortf(exitcode.ErrIllegalState, "proving period at %d elapsed, next one not yet opened", deadline.PeriodStart)
+		currDeadline := st.DeadlineInfo(currEpoch)
```

Comment:
This delta is just re-arrangement to make this case follow the same pattern as the others. No behaviour change intended. The `deadline.PeriodElapsed` check has moved inside `detectFaultsThisPeriod`